### ISSUE TITLE
Playlists via playify

### ISSUE THREAD

**Issue by GreenDiscord** [retromp3/retro#22]
Just saw a PR that was completed [Here](https://github.com/iberatkaya/playify/pull/5), It seem's playify now supports playlist.
I can attempt to make a PR to implement playlists into Retro, if it's not already in the works.

**Comment by sakofchit** [comment]
Yeah that'd be amazing––although Playify's implementation in Retro isn't complete either.

**Comment by GreenDiscord** [comment]
Alright, I'll just have to find where I put my mac..
